docs/data.md

```diff
@@ -24,6 +24,31 @@ the level and type of the log is listed.
 
 ### `merino-web`
 
+- `INFO web.suggest.request` - A suggestion request is being processed. This
+  event will include fields for all relevant details of the request. **Fields:**
+
+  - `query` - If query logging is enabled, the text the user typed. Otherwise
+    and empty string.
+  - `country` - The country the request came from.
+  - `region` - The first country subdivision the request came from.
+  - `city` - The city the request came from.
+  - `dma` - A US-only location description that is larger than city and smaller
+    than states, but does not align to political borders.
+  - `agent` - The original user agent.
```

Comment:
I don't see the full agent included in the implementation of the log message. Unless we have a strong argument for why we need to include it, raw user agent is sensitive enough that it should not be logged along with queries; we should only keep the fields parsed out of it.

Reply:
User agent is also something that comes from the MozLog middleware. We could change it if needed (Merino is currently the only user of that library). Use agent is included by the library because MozLog recommends it.

Reply:
Adding a bit here, the UA data is "smoothed" via `DeviceInfo`, which strips and normalizes a lot of the UA info into [buckets](https://github.com/mozilla-services/merino/blob/dafd421b55f8ca444022e9f935d8ce7c5a44573a/merino-web/src/extractors.rs#L232-L240) Generally, we don't record OS version info or much of the other data in the UA.

Reply:
I am fine with the `os_family`, `form_factor`, and `browser` parsed from the UA that appear below. Are those fields what `DeviceInfo` produces, or is there an overall smoothed UA string that's produced as this `agent` field?

Reply:
The `agent` field isn't smoothed; it's the original value. I'll file follow up issues to modify the mozlog middleware.

Reply:
https://github.com/mozilla-services/common-rs/issues/29